spec/services/push_notifications/send_spec.rb

```diff
@@ -55,4 +64,28 @@
         .and change(PushNotifications::DeliverWorker.jobs, :size).by(1)
     end
   end
+
+  context "with devices for multiple users" do
+    before do
+      allow(FeatureFlag).to receive(:enabled?).with(:mobile_notifications).and_return(true)
+      allow(ApplicationConfig).to receive(:[]).with("RPUSH_IOS_PEM").and_return("dGVzdGluZw==")
+      allow(ApplicationConfig).to receive(:[]).with("COMMUNITY_NAME").and_return("Forem")
```

Comment:
*nonblocking*

Would it make sense to do something like this?

```suggestion
      stub_const "ApplicationConfig", {
        "RPUSH_IOS_PEM" => Base64.encode64("testing"),
        "COMMUNITY_NAME" => "Forem",
      }
```

Specifically, I find the stubbing of `ApplicationConfig#[]` pretty difficult to follow. If `ApplicationConfig` is used with a hash, we can stub it with one — I find that easier to read. It's possible that that's just because I'm not used to the existing way, but like I mentioned above, I like to use my confusion as a guide in code review. 🙂

Also, the suggested change includes replacing the hardcoded base-64 string with an expression that indicates what is expected there. I had to run it through `Base64.decode` in order to figure out what the purpose of the hardcoded string was.

Reply:
I understand the reason why this can be a little confusing and I don't have strong opinions on it (I could live with it either way).

For this PR I'll avoid adding this change mainly because we do have plenty of occurrences (37 from a quick search) that use the current syntax to stub `ApplicationConfig`, and this would break that consistency.

If we can make a point with the rest of the team to migrate all of them in a separate PR and maintain that standard going forward that would be awesome 🙂

Reply:
I think overall we need a better way to stub out values in ApplicationConfig: a lot of tests are doing it in different ways, and most of the time only one of two values need to be set in a test (and we want the rest of the values to be defaulted to the rest of the application still works: this is especially true in request tests).